client/app/components/sections/Topbar/Topbar.story.js

```diff
@@ -68,7 +72,9 @@ storiesOf('Top bar')
         image: 'https://s3.amazonaws.com/sharetribe-manual-assets/styleguide/bikerrrs-logo.png',
         image_highres: 'https://s3.amazonaws.com/sharetribe-manual-assets/styleguide/bikerrrs-logo.png',
       },
-      search_mode: 'keyword-and-location',
-      search_keyword_placeholder: 'Search...',
-      search_location_placeholder: 'Location',
+      search: {
+        mode: 'keyword-and-location',
+        keyword_placeholder: 'Search...',
+        location_placeholder: 'Location',
```

Comment:
Maybe this would be a good spot to convert these to camelcase, and simplify the props-passing part in `Topbar`?